src/wtf/utils.cc

```diff
@@ -203,18 +203,23 @@ bool SanitizeCpuState(CpuState_t &CpuState) {
   // Validate that the Reserved field of each segment contains bits 16-19 of the
   // Limit
   //
+  
 
-  Seg_t *Segments[] = {&CpuState.Es, &CpuState.Fs, &CpuState.Cs,
-                       &CpuState.Gs, &CpuState.Ss, &CpuState.Ds};
+  #if ELF_COMPILATION != 1
```

Comment:
Okay, I think we shouldn't need this. Either there is a bug in my sanitization below, or there is something weird going on in Linux's segment registers. Either or, it's something that should probably fixed somewhere else.

Do you have more details on this? Like which register fails the sanitization and what is  their values? Happy to investigate.